### PR TITLE
Add github corner

### PR DESCRIPTION
**Pull request by adamsandle**

### CODE REVIEW

**Review by raccube — APPROVED** [review]
Looks good 👍

File: src/App.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ class App extends Component {
             <input type="button" onClick={this.toggleShowAnswer} ref={(back) => {this.back = back;}} value="Back"/>
           </div>
         }
+        <GithubCorner href="https://github.com/hoily/how-do-i" bannerColor={"#ffffff"} octoColor={"#222"} />
```
Comment (RealOrangeOne) [inline src/App.js]: Curley brackets aren't needed on string-literal props

**Comment by adamsandle** [comment]
@RealOrangeOne Made requested changes

**Review by RealOrangeOne — APPROVED** [review]

**Review by sedders123 — APPROVED** [review]
LGTM

**Pull request merged**